lru_cache.py

```diff
@@ -1,8 +1,8 @@
-class LRUCache(object):
+class LRUCache():
     def __init__(self, capacity):
         self.capacity = capacity
-        self.cache = {}
         self.lru = {}
+        self.cache = {}
```

Comment:
Here instead of using two different dictionaries, its better to have class object for multiple properties and store using that object  in to one dictionary.